docs/CLI.md

```diff
@@ -301,6 +301,10 @@ Lets you specify a custom test runner.
 
 Lets you specify a custom test sequencer. Please refer to the documentation of the corresponding configuration property for details.
 
+### `--testTimeout=<number>`
+
+Default timeout of a test in milliseconds.
```

Comment:
Should we say it's `5000` if not passed?

Reply:
5000 the current default value. With this PR it will not change, but I will add the default value. It much clear